OpenRA.Game/Graphics/TargetLineRenderable.cs

```diff
@@ -47,7 +47,8 @@ public void Render(WorldRenderer wr)
 			var a = first;
 			foreach (var b in waypoints.Skip(1).Select(pos => wr.ScreenPxPosition(pos)))
 			{
-				Game.Renderer.WorldLineRenderer.DrawLine(a, b, color);
+				if (Game.Settings.Game.DrawTargetLine)
+					Game.Renderer.WorldLineRenderer.DrawLine(a, b, color);
 				wr.DrawTargetMarker(color, b);
```

Comment:
I agree. In cnc you get just a coloured dot at the destination with this, which looks kinda weird.